torch/csrc/distributed/autograd/context/dist_autograd_container.h

```diff
@@ -18,12 +18,18 @@ namespace autograd {
 // autograd_context_id. The autograd_context_id itself is a 64 bit globally
 // unique id. The first 16 bits is the worker_id and the next 48 bits is an
 // auto-incrementing id for each worker.
-class DistAutogradContainer {
+//
+// This container is also responsible for maintaining a globally unique message
```

Comment:
Just to confirm, this will be independent from the `Message.id()` we used to match RPC request/response, right?